src/main/python/merchant_comparison.py

```
@@ -0,0 +1,1734 @@
+                widget.configure(bg=theme['button_bg'], fg=theme['button_fg'])
+            elif widget_class == 'Text':
+                widget.configure(bg=theme['text_bg'], fg=theme['text_fg'], 
+                               insertbackground=theme['fg'])
+            elif widget_class == 'Checkbutton':
+                widget.configure(bg=theme['frame_bg'], fg=theme['fg'], 
+                               selectcolor=theme['entry_bg'])
+            elif widget_class == 'Scale':
+                widget.configure(bg=theme['bg'], fg=theme['fg'])
+            elif widget_class == 'Canvas':
+                widget.configure(bg=theme['bg'])
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)